src/pages/_error.tsx

```diff
@@ -1,52 +1,41 @@
 import React, { ReactElement } from 'react';
-import { NextPageContext } from 'next';
 import Layout from '../layout/Layout';
+import { FEEDBACK_LINK } from '../constants';
 
 const title = 'Error - Fares data build tool';
 const description = 'Error page of the Fares data build tool';
 
-interface ErrorProps {
-    statusCode: number;
-}
-
-const Error = ({ statusCode }: ErrorProps): ReactElement => (
+const Error = (): ReactElement => (
     <Layout title={title} description={description}>
         <main className="govuk-main-wrapper app-main-class" id="main-content" role="main">
             <div className="govuk-grid-row">
-                {!statusCode || statusCode !== 404 ? (
+                <div className="govuk-grid-column-two-thirds">
                     <div>
                         <h1 className="govuk-heading-xl">Sorry, there is a problem with the service.</h1>
                         <p className="govuk-body">Try again later.</p>
-                        <p className="govuk-body">Your answers have not been saved, click continue to start again.</p>
                         <p className="govuk-body">
-                            Contact us with the feedback button at the top of this page for assistance.
+                            Your answers have not been saved, use the button below to start again.
+                        </p>
+                        <p className="govuk-body">
+                            {' '}
+                            <a className="govuk-link" id="feedback_link" href={FEEDBACK_LINK}>
+                                Contact
+                            </a>{' '}
+                            us for assistance.
                         </p>
                     </div>
-                ) : (
-                    <div>
-                        <h1 className="govuk-heading-xl">Page not found</h1>
-                        <p className="govuk-body">If you entered a web address, please check it was correct.</p>
-                        <p className="govuk-body">Click continue to start again.</p>
-                    </div>
-                )}
-
+                </div>
                 <br />
-                <a
-                    href="operator"
-                    role="button"
-                    draggable="false"
-                    className="govuk-button govuk-button--start"
-                    data-module="govuk-button"
-                >
-                    Continue
+                <a href="operator" role="button" draggable="false" className="govuk-button" data-module="govuk-button">
```

Comment:
This and the `<br />` need to be in the `govuk-grid-column-two-thirds` div above, it's moved into its own column

Reply:
Also we should use /operator here as well